### PR TITLE
Rc68 without pr13162

### PR DESCRIPTION
**Pull request by AndrewMeadows**
* removed fix for "collisionless avatars" in attempt to fix a mysterious occasional crash on Mac

### CODE REVIEW

**Comment by AndrewMeadows** [comment]
This is a test PR:  RC68 with PR-13162 removed but including our fix for tiny masses.  If this manages to fix the mysterious Mac crash then great.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10529/HighFidelity-Beta-PR13352-d31ee52.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10529/HighFidelity-Beta-PR13352-d31ee52.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10529/HighFidelity-Beta-Interface-PR13352-d31ee52.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10529/HighFidelity-Beta-PR13352-d31ee52.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10529/HighFidelity-Beta-Interface-PR13352-d31ee52.exe

**Comment by qlimnavarro** [comment]
Currently Testing

File: interface/src/avatar/AvatarMotionState.cpp [hunk 1]
```diff
@@ -74,31 +60,25 @@ void AvatarMotionState::getWorldTransform(btTransform& worldTrans) const {
     worldTrans.setRotation(glmToBullet(getObjectRotation()));
     if (_body) {
         _body->setLinearVelocity(glmToBullet(getObjectLinearVelocity()));
-        _body->setAngularVelocity(glmToBullet(getObjectAngularVelocity()));
+        _body->setAngularVelocity(glmToBullet(getObjectLinearVelocity()));
```
Comment (sethalves) [inline interface/src/avatar/AvatarMotionState.cpp]: is this right?  set Angular from Linear?

File: interface/src/avatar/AvatarMotionState.cpp [hunk 2]
```diff
@@ -74,31 +60,25 @@ void AvatarMotionState::getWorldTransform(btTransform& worldTrans) const {
...
-        _body->setLinearVelocity(velocity);
-        _body->setAngularVelocity(glmToBullet(getObjectAngularVelocity()));
-    }
+    btVector3 targetPosition = glmToBullet(getObjectPosition());
+    btTransform newTransform;
+    newTransform.setOrigin((1.0f - tau) * currentPosition + tau * targetPosition);
+    newTransform.setRotation(glmToBullet(getObjectRotation()));
+    _body->setWorldTransform(newTransform);
+    _body->setLinearVelocity(glmToBullet(getObjectLinearVelocity()));
+    _body->setAngularVelocity(glmToBullet(getObjectLinearVelocity()));
```
Comment (sethalves) [inline interface/src/avatar/AvatarMotionState.cpp]: here also?

**Comment by qlimnavarro** [comment]
Pass @conklin94122

**Pull request merged**